regress/copy-mode-test-emacs.sh

```diff
@@ -42,7 +42,18 @@ $TMUX send-keys -X begin-selection
 $TMUX send-keys -X next-word-end
 $TMUX send-keys -X next-word-end
 $TMUX send-keys -X copy-selection
-[ "$($TMUX show-buffer)" = "$(echo -e "words\n        Indented")" ] || exit 1
+if [ "$($TMUX show-buffer)" != "$(printf "words\n        Indented")" ] 
```

Comment:
Can you please do this for this entire file where it uses echo -e? (ex. line 52)

Reply:
fix